felix/bpf/counters/counters.go

```diff
@@ -155,6 +158,18 @@ var descriptions DescList = DescList{
 		Counter:  DroppedBlackholeRoute,
 		Category: "Dropped", Caption: "packets hitting blackhole route",
 	},
+	{
+		Counter:  SourceCollision,
+		Category: "Other", Caption: "packets hitting NAT source collision",
+	},
+	{
+		Counter:  ConntrackCreateFailed,
+		Category: "Dropped", Caption: "failed to create conntrack",
+	},
+	{
+		Counter:  SourceCollisionFailed,
+		Category: "Dropped", Caption: "packets hitting NAT source collision failed",
```

Comment:
Can the Caption for SourceCollision and SourceCollisionFailed be better to capture the difference between these 2? Its a bit confusing. Maybe something like `packets hitting NAT source collision unresolved`.